region/lowernorfair/west.json

```diff
@@ -94,13 +119,31 @@
           "to": [
             {
               "id": 1,
-              "requires": [ "UsedAcidChozoStatue" ],
-              "unlock": null,
-              "note": "Because going to the statue without unlocking it is pointless, this path will assume the acid has been cleared"
```

Comment:
Any reason the note was removed? The flag it explained is still in the requirements.

Reply:
Done in the middle of the process where things where up in the air. Putting it back.